doc/CCPPtechnical/source/ConstructingSuite.rst

```diff
@@ -8,14 +8,14 @@ Constructing Suites
 Suite Definition File
 ==============================
 
-The :term:`SDF` is a file in XML format used to specify the name of the suite, the physics schemes to run, groups of physics that run together, the order in which to run the physics, and whether subcycling will be used to run any of the parameterizations with shorter timesteps. The :term:`SDF` files are located in ``ccpp/suites/suite_*.xml``. 
+The :term:`SDF` is a file in XML format used to specify the name of the suite, the physics schemes to run, groups of physics that run together, the order in which to run the physics, and whether subcycling will be used to run any of the parameterizations with shorter timesteps. The :term:`SDF` files are located in ``ccpp/suites/suite_*.xml``.
```

Comment:
Are you OK with "The :term:`SDF` files are part of the host model code"?

Reply:
We plan to store them in our physics library.
For now, your suggestion might be okay because we do not have the ability to write host-model portable suites. Make your suggested change and let's keep an eye on it.